data/catalogs/deltares_data.yml

```diff
@@ -1782,14 +1931,19 @@ vito_2015_v2.0.2:
     paper_doi: 10.5281/zenodo.3939038
     paper_ref: Buchhorn et al (2020)
     source_url: https://land.copernicus.eu/global/products/lc
-    source_version: v2.0.2
+    source_spatial_extent:
+      West: -180.0
+      South: -60.0
+      East: 180.0
+      North: 80.0
+  version: 2.0.2
   path: landuse/vito/ProbaV_LC100_epoch2015_global_v2.0.2_discrete-classification_EPSG-4326.tif
 
-vito_2016_v3.0.1:
+vito_2016:
```

Comment:
Add a yr placeholder for 2016 to 2019?
Only diff is lowercase on discrete_classication for 2019 compared to the other years

Reply:
I think it is going to get confusing. Vito_2015 is of version 2.0.2 while the rest of the vito datasets are of version 3.0.1, so I can't group all the Vito datasets with a placeholder.